src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+    private final int index = random.nextInt(Color.values().length);
```

Comment:
It's better to move this logic to method, there is no need to define index on class level